CountOnMe/Model/CalculatorManager.swift

```diff
@@ -55,27 +55,34 @@ struct CalculatorManager {
     }
 
     private func formatString(_ unformattedString: String) -> String? {
-        let numberFormatter: NumberFormatter = .init()
-        numberFormatter.maximumFractionDigits = 4
-        numberFormatter.locale = .current
-        numberFormatter.decimalSeparator = "."
-        guard let formattedString = numberFormatter.string(from: NSDecimalNumber(string: unformattedString)) else {
+        let formatter = NumberFormatter.shared
+        guard let formattedString = formatter.string(from: NSDecimalNumber(string: unformattedString)) else {
             return nil
         }
         return formattedString
     }
 }
 
 enum CalculationError: Error {
-    case unvalidOperator
+    case invalidOperator
     case divideByZero
 
     var message: String {
         switch self {
-        case .unvalidOperator:
+        case .invalidOperator:
             "unvalid operator"
         case .divideByZero:
             "division by 0"
         }
     }
 }
+
+extension NumberFormatter {
```

Comment:
**suggestion (blocking)**: Since it is a global extension (not a delegate adoption) it should be moved to its own file.

`$(PROJECT_DIR)/Extensions/NumberFormatterExtensions` (or similar)